service/internal/bootstrap/orgcluster.go

```diff
@@ -0,0 +1,102 @@
+// Package bootstrap ensures certain RBAC resources like ClusterRoles,
+// ClusterRoleBindings, and ServiceAccounts. If they exist, they will
+// be modified according to spec. If not, they are created.
+package bootstrap
```

Comment:
The bootstrap package is a bit of a mess, so i decided to add the definitions for the org cluster apps clusterRoles in this extra file.